src/app/dialog/dialog-routing.module.ts

```diff
@@ -105,7 +105,7 @@ const routes: Routes = [
     outlet: 'dialog',
   },
   {
-    path: `${DialogPath.CALENDAR_EVENT}/:calendarId/:time/:documentId`,
+    path: `${DialogPath.CALENDAR_EVENT}/:calendarId/:time/:documentId/:stemIndex`,
```

Comment:
After new changes, we can have same query stems in query. In this case I need to know which config I should use to display and edit data in dialog, because `collectionId` is not enough.